docs/api/desktop-capturer.md

```diff
@@ -100,8 +100,23 @@ Returns `Promise<DesktopCapturerSource[]>` - Resolves with an array of [`Desktop
 [`navigator.mediaDevices.getUserMedia`]: https://developer.mozilla.org/en/docs/Web/API/MediaDevices/getUserMedia
 [`systemPreferences.getMediaAccessStatus`]: system-preferences.md#systempreferencesgetmediaaccessstatusmediatype-windows-macos
 
-## Caveats
+## System Audio Capture (macOS 13+)
 
-`navigator.mediaDevices.getUserMedia` does not work on macOS for audio capture due to a fundamental limitation whereby apps that want to access the system's audio require a [signed kernel extension](https://developer.apple.com/library/archive/documentation/Security/Conceptual/System_Integrity_Protection_Guide/KernelExtensions/KernelExtensions.html). Chromium, and by extension Electron, does not provide this.
+ On macOS 13+, `navigator.mediaDevices.getDisplayMedia` can be used to capture system audio
+ if the [`setDisplayMediaRequestHandler`](./session.md#sessetdisplaymediarequesthandlerhandler-opts) `callback` 
+ is passed a stream that has `audio: 'loopback'`. See the [example](./desktop-capturer.md) above.
```

Comment:
```suggestion
 is passed a `streams` object that has `audio: 'loopback'`. See the example above.
```

Two changes:

* "`streams` object"
* I'd remove the link because we don't have an anchor tag to link to